kerl

```diff
@@ -937,6 +963,69 @@ ACTIVATE_CSH
     echo "kerl_deactivate"
 }
 
+install_docsh() {
+    # TODO: SHOULD check version compatibility
```

Comment:
The CI runs only for 18 and 19. I'm not sure if currently things would break with prior versions or not.